src/dvclive/live.py

```diff
@@ -456,6 +460,8 @@ def log_artifact(
                         name,
                     )
 
+        self._outs[str(path)] = cache
```

Comment:
Shouldn't this be nested inside the if?

Reply:
🤔 Not sure. Any reason not to always add it? No strong opinion.

Reply:
If we do not enter the if we are not doing anything inside the method